src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1516,6 +1516,16 @@ qlonglong TorrentImpl::nextAnnounce() const
     return lt::total_seconds(m_nativeStatus.next_announce);
 }
 
+// in order to produce floating-point numbers using `std::chrono::duration_cast`,
+// we should use `qreal` as `Rep` to define the `months` duration
+using months = std::chrono::duration<qreal, std::chrono::months::period>;
```

Comment:
still, I would suggest moving these inside the function since they are used locally and it isn't too useful anywhere else.

Reply:
I've moved these lines as you suggested.